src/components/interactive/Toast.jsx

```diff
@@ -124,10 +125,10 @@ const Toast = ({
 
 Toast.propTypes = {
   /** JSX Content of Toast */
-  content: PropTypes.element.isRequired,
+  content: PropTypes.node.isRequired,
 
   /** Type of toast */
-  type: PropTypes.oneOf(['info', 'warn', 'error', 'success', 'progress']).isRequired,
+  type: PropTypes.oneOf(TYPES),
```

Comment:
So one thing I noticed when I played with Toast, is there were bunches of times where it would display weird, because I forgot to pass it a type.

So the idea I had, was to specific a default to always render:

```
const Toast = ({
  type = 'info',
```

That way, there isn't a way the consumer can mess it up. It'll display as info, or as another type if they have to pass it in.

Thoughts?

Reply:
Ah, the default makes sense, I somehow missed that. Let's keep it